liason-guide.md

```diff
@@ -0,0 +1,51 @@
+# TOC Liaison Guide
+
+As a TOC Liaison, your role is to facilitate direct support from the TOC to your assigned projects.
+
+## Liason Responsibilities
+
+- **Monthly Meetings**: Meet with project leads at least once a month to gather updates and discuss any obstacles.
+- **Quarterly Reporting**: Submit a summary of project status to the TOC Chair for inclusion in the quarterly board report.
+- **Issue Escalation**: Privately escalate major issues or complaints to the TOC unless the project requests otherwise.
+- **Project Growth Track**: Ensure that all projects understand the path to [promotion](https://community.finos.org/docs/governance/Software-Projects/project-lifecycle) or [archival](https://community.finos.org/docs/governance/software-projects/stages/archived/).
+
+### Liason Report Template
+
+Please use the following template to streamline consolidation into the monthly newsletter and quarterly report for the FINOS Governing Board.
+
+```markdown
+## [Enter project name]
+
+Liaison: [Your name]
+Leads: [Project leads]
+
+### Progress
+
+- Significant milestones achieved
+- Notable contributions or updates
+
+### Challenges
+
+- Any obstacles or issues raised by the project
+- Any support requested from the TOC
+
+### Next Steps
+
+- Planned activities or upcoming milestones
+- Suggested actions to address challenges
+
+### Additional Notes
+
+- Any further observations or recommendations
+```
+
+## How to meet with your projects
+
+- **Set aside time**: Create a recurring monthly meeting for 60 minutes, though you may not need the whole hour every time.
+- **Solicit Updates**: Ask how things are going, what the team is working on, and whether there are any obstacles. Follow up on previous discussions. Take notes for later!
+- **Share Knowledge**: If there is interest, volunteer news from across the landscape. This may be your project leads' only practical opportunity to learn about other ongoing efforts!
```

Comment:
We should somehow collate and track the items that would worth sharing and find a way to surface them up at the right time. I was thinking maintaining a 4 column google sheet, landscape item A, landscape item B, information, whether it has been communicated. This information later can be resurfaced other forms too.